src/input/mod.rs

```diff
@@ -1000,7 +1000,7 @@ impl State {
                     .cloned();
                 if let Some(seat) = maybe_seat {
                     self.common.idle_notifier_state.notify_activity(&seat);
-                    if event.fingers() >= 3 {
+                    if event.fingers() >= 3 && workspace_overview_is_open(&seat.active_output()) {
```

Comment:
Isn't that reversed? If we have a gesture with three or more fingers, we handle it in the compositor. So when the workspace overview is **not** open, we want to skip this behaviour and send the gesture to the focused application. (The overview.)

Reply:
Ugh, I actually had fix this already when I tested it but hadn't amended the commit.